src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,6 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+public class AuthenticationService extends UserService {
```

Comment:
This is very bad practice. Please do not inherit services. We will talk about it next week